src/tables/sortable-table.js

```diff
@@ -148,41 +151,49 @@ function SortableTable({
   }
 
   return (
-    <table className={classnames(className, { 'sortable-table': !disableSort })} {...filterInvalidDOMProps(rest)}>
-      <thead>
-        <tr>
-          {columns.map((column, key) => {
-              const Header = column.headerComponent || headerComponent || DefaultHeader
+    <>
+      {caption && <caption>{caption}</caption>}
```

Comment:
this is the only change in return

Reply:
This needs to be rendered as the first child of the `<table>`. See previous comment for source